controllers/semeru_compiler.go

```diff
@@ -109,10 +132,24 @@ func (r *ReconcileWebSphereLiberty) reconcileSemeruDeployment(wlva *wlv1.WebSphe
 					Env: []corev1.EnvVar{
 						{Name: "OPENJ9_JAVA_OPTIONS", Value: "-XX:+JITServerLogConnections"},
 					},
+					LivenessProbe:  &livenessProbe,
+					ReadinessProbe: &readinessProbe,
 				},
 			},
 		},
 	}
+
+	// Copy the pullSecret from the WebSphereLibertyApplcation CR
+	if wlva.Spec.PullSecret != nil {
+		deploy.Spec.Template.Spec.ImagePullSecrets = []corev1.LocalObjectReference{{
+			Name: *wlva.Spec.PullSecret,
+		}}
+	}
+
+	// Copy the securityContext from the WebSphereLibertyApplcation CR
+	if wlva.Spec.SecurityContext != nil {
```

Comment:
There are some security contexts we set explicitly for the default scenario. We should do the same as app ([here](https://github.com/application-stacks/runtime-component-operator/blob/9a47892d9c99f2531600f50e1cf6ce7191e844d0/utils/utils.go#L658))